docs/proposals/PROPOSAL_A.md

```diff
@@ -0,0 +1,293 @@
+# Proposal A
+
+This proposal defines a new artifact manifest and corresponding referrers extension API for registries to support reference types.
```

Comment:
The entire section should be exactly just "Defines a new artifact manifest and corresponding referrers extension API"

"The modifications include.." is unneeded or can be baked into the Description section

Reply:
Please take a moment to review the template: https://github.com/opencontainers/wg-reference-types/blob/main/docs/TEMPLATE.md#proposal-letter